docs/TestCoverage.md

```diff
@@ -333,6 +357,30 @@ result = argmin_use_numpy(data, axis=axis, keepdims=keepdims)
 expect(node, inputs=[data], outputs=[result], name='test_argmin_keepdims_random')
 ```
 
+</details>
+<details>
+<summary>negative_axis_keepdims</summary>
+
+```python
+data = np.array([[2, 1], [3, 10]], dtype=np.float32)
+axis = -1
+keepdims = 1
+node = onnx.helper.make_node(
+    'ArgMin',
+    inputs=['data'],
+    outputs=['result'],
+    axis=axis,
+    keepdims=keepdims)
+# result: [[1], [0]]
+result = argmin_use_numpy(data, axis=axis, keepdims=keepdims)
```

Comment:
>argmin_use_numpy [](start = 9, length = 16)

Your PR is already great. Just wondering if you can append something like
```
# The content of result is
# [1, 2, 3, 4]
```
to be more specific to the reader.